src/danog/MadelineProto/Db/NullCache/NullCacheTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace danog\MadelineProto\Db\NullCache;
+
+trait NullCacheTrait
```

Comment:
Maybe its better to add some property-flag to ArrayCacheTrait, to control caching. 
And we can change $propertyType from string to array with name and configs for each property.

Reply:
This can be done, but by using a NullCacheTrait we're actually doing JS-style composition, completely avoiding run-time ifs)))